ovos_utils/messagebus.py

```diff
@@ -135,10 +144,19 @@ def get_websocket(host, port, route='/', ssl=False, threaded=True):
     return client
 
 
-def get_mycroft_bus(host='0.0.0.0', port=8181, route='/core', ssl=False):
+def get_mycroft_bus(host: str = None, port: int = 8181, route: str = '/core',
```

Comment:
all params should be made None by default and come from config

Reply:
Ah, missed that while I was refactoring